preset/README.md

```diff
@@ -0,0 +1,41 @@
+# Babel Preset for Source Code
+
+> A Babel 6 preset that compiles official ES2015+ down to ES5 by automatically determining the Babel plugins and polyfills you need based Salesforce's supported browsers. This includes the Design System for React project.
```

Comment:
> based Salesforce's

based on Salesforce's

Reply:
This may be more clear:

A Babel 6 preset that makes DSR compatible with Salesforce's supported browsers. You need this to transpile the source code for it to be readable by a module bundler (ie: Webpack).